pawn/Features/Gameplay/Vehicles/Cruise/CruiseCommands.pwn

```diff
@@ -91,6 +91,11 @@ class CruiseCommands {
         if (cruiseLeaderId == Player::InvalidId)
             return 1;
 
+        if (cruiseLeaderId == IsPlayerNPC(cruiseLeaderId)) { 
```

Comment:
It compiles, but has two issues:
  * ID `0` cannot be cruise leader _unless_ they're an NPC,
  * ID `1` can only be cruise leader when they _are_ an NPC.

The condition you're looking for it this:

``` javascript
if (IsPlayerNPC(cruiseLeaderId)) {
```

As [the documentation](https://wiki.sa-mp.com/wiki/IsPlayerNPC) explains, this returns `0` when the given `playerid` is a player, or `1` when the given `playerid` is an NPC. The case will happen when the value is _truthy_, which means `1`.